### ISSUE TITLE
Season reset to fall when server reboots with custom season length defined

### ISSUE THREAD

**Issue by cherrygarcia881** [shudnal/Seasons#21]
Hi, I defined a custom season length for each season as 15 days. When it was in spring day 10 I needed to reboot the server. Upon rebooting it reset the current season to fall day 10. I have seen this happen a few times. My best guess is it is related to fall.json being the first file alphabetically. As a work around, is there a console command to manually change the current season?

Great mod by the way!

**Comment by shudnal** [comment]
Fire order doens't mean anything. Do you have error in log file? Season and day value depends on current world day. Are you sure world day didn't change between sessions?

You can override current season using general mod config. **Season - override** section. It is there mostly for debugging. It's better to figure out the reason.

You can enable logging and see if there something suspicious about Seasons output. And if you manage to reproduce the issue the logs before and after rebooting could help.

At first you can just use console command **time** it will show you current world day. Then check if it was changed after reboot,
If it is the same after reboot but season or day was changed then there is something on Seasons side.

In short you can define current Season and day by simple algorithm.
Let's say the world day is 354. Every season length is 15. Then year length is 15 * 4 = 60 days.
354 % 60 = 54 (Euclidean division – or division with remainder)
54 = 15(spring) + 15(summer) + 15(fall) + 9 days of winter. It should be Winter day 9.

**Comment by cherrygarcia881** [comment]
Hmm, to be honest I can't explain what it was doing. I just messed with the settings some more and it synched to the correct day. Maybe the days just got temporarily out of synch, I dunno.

**Comment by shudnal** [comment]
Anyway if you still have any questions I'm here it answer.
